setup/context/commands.go

```diff
@@ -63,17 +65,9 @@ func Apply(c *Context, force bool, dockercfg string) error {
 		}
 	}
 
-	if dockercfg != "" {
-		fmt.Printf("Detected dockercfg: `%s`\n", dockercfg)
-		instancePath := fmt.Sprintf("%s/dockercfg", c.InstanceDir)
-
-		if err := CopyFile(dockercfg, instancePath); err != nil {
-			return fmt.Errorf("Failed to copy dockercfg: %v.", err)
-		}
-	}
-
-	// validate dockercfg; if file does not exist, write empty dockercfg file
-	if err := checkDockercfg(c, force); err != nil {
+	// Check for docker config file (config.json or dockercfg) and copy it to
+	// instance directory in `dockercfg` format.
+	if err = checkDockercfg(c, dockerCfgDir, force); err != nil {
```

Comment:
@jonathanstallings I think zpatrick is just saying that passing `force` as a param/behavior around to other functions, even if they are helpers, is not ideal.